spec/models/user_spec.rb

```diff
@@ -55,4 +55,19 @@
       expect(user1).to be false
     end
   end
+
+  describe 'review notification' do
+    let(:user) { create(:user) }
+
+    before do
+      @card = user.cards.create(original_text: 'hello', translated_text: 'Привет',
```

Comment:
Line is too long. [82/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.